sdk/search/CONTRIBUTING.md

```diff
@@ -72,57 +72,54 @@ live tests against a pull request by commenting `/azp run net - search - tests`
-}
-
-PS C:\src> az group create --location <location> --name <resource_group>
-{
-  "id": "/subscriptions/<subscription_id>/resourceGroups/<resource_group>",
-  "location": <location>,
-  "managedBy": null,
-  "name": "<resource_group>",
-  "properties": {
-    "provisioningState": "Succeeded"
-  },
-  "tags": null,
-  "type": "Microsoft.Resources/resourceGroups"
-}
-
-PS C:\src> setx AZURE_TENANT_ID <tenant_id>
-SUCCESS: Specified value was saved.
-
-PS C:\src> setx AZURE_CLIENT_ID <client_id>
-SUCCESS: Specified value was saved.
-
-PS C:\src> setx AZURE_CLIENT_SECRET <client_secret>
-SUCCESS: Specified value was saved.
-
-PS C:\src> setx AZURE_SUBSCRIPTION_ID <subscription_id>
-SUCCESS: Specified value was saved.
-
-PS C:\src> setx AZURE_RESOURCE_GROUP <resource_group>
-SUCCESS: Specified value was saved.
-
-PS C:\src> setx AZURE_LOCATION <location>
-SUCCESS: Specified value was saved.
+PS C:\src> Connect-AzAccount -Subscription 'YOUR SUBSCRIPTION ID'
+PS C:\src> $sp = New-AzADServicePrincipal -Role Owner
+PS C:\src> eng\common\TestResources\New-TestResources.ps1 `
+  -BaseName 'myusername' `
+  -ServiceDirectory search `
+  -TestApplicationId $sp.ApplicationId `
+  -TestApplicationSecret (ConvertFrom-SecureString $sp.Secret -AsPlainText)
+```
+
+Along with some log messages, this will output environment variables based on your
+current shell like in the following example:
+
+```
+$env:AZURE_TENANT_ID = '04acef35-c7bd-4d14-bfd9-59f11b7b9eac'
+$env:AZURE_CLIENT_ID = 'ce1a3a01-424f-4e34-b9a6-823e2b1ae783'
+$env:AZURE_CLIENT_SECRET = 'c27ccc92-e1ca-4d29-8f4e-c6a1ee61ff57'
+$env:AZURE_SUBSCRIPTION_ID = 'd686c17c-aade-4238-bce0-290453cfcf97'
+$env:AZURE_RESOURCE_GROUP = 'rg-myusername'
+$env:AZURE_LOCATION = 'westus2'
+$env:AZURE_SEARCH_STORAGE_NAME = 'myusernamestg'
+$env:AZURE_SEARCH_STORAGE_KEY = 'Of2O5Snep5tl13bfjh02/fSNYfrBPXV7CYK7EVnMm/z9fN7zCcq6WKuWfZDM9QsTORvC7zYLifyIEtymI5VCmA=='
+```
+
+For security reasons we do not set these environment variables automatically for either
+the current process or persistently for future sessions. You must do that yourself.
+
+If your current shell was detected propertly, you should be able to copy and paste the
+output directly in your terminal and add to your profile script. For example,
+in PowerShell on Windows you could copy and paste the output and run the following command:
+
+```powershell
+$env:AZURE_TENANT_ID = '04acef35-c7bd-4d14-bfd9-59f11b7b9eac'
+$env:AZURE_CLIENT_ID = 'ce1a3a01-424f-4e34-b9a6-823e2b1ae783'
+$env:AZURE_CLIENT_SECRET = 'c27ccc92-e1ca-4d29-8f4e-c6a1ee61ff57'
+$env:AZURE_SUBSCRIPTION_ID = 'd686c17c-aade-4238-bce0-290453cfcf97'
+$env:AZURE_RESOURCE_GROUP = 'rg-myusername'
+$env:AZURE_LOCATION = 'westus2'
+$env:AZURE_SEARCH_STORAGE_NAME = 'myusernamestg'
+$env:AZURE_SEARCH_STORAGE_KEY = 'Of2O5Snep5tl13bfjh02/fSNYfrBPXV7CYK7EVnMm/z9fN7zCcq6WKuWfZDM9QsTORvC7zYLifyIEtymI5VCmA=='
+
+dir env:AZURE* | % { setx $_.Name $_.Value }
```

Comment:
Let's not do this.  I don't want to give users commands that might cause configuration weirdness.  I'd prefer to just give them a list of setx commands to paste.

Reply:
This was decided for security reasons. The scripts don't actually set the environment variables, and it has to be cross-platform. setx is Windows. On other platforms, persisting variables across sessions (like in .bashrc, .profile, etc.) varies from shell to shell, or even with customizations. That's another reason we don't do it.

Reply:
I'm not talking about the script.  I'm talking about your example here.  It's fine to have a bunch of `setx` examples.  I don't think we should `setx` every AZURE env var in your current process.  I think this .md file should delete line 122.

Reply:
Then where do you suggest people persist them? If they lose them, they are gone forever. Remember, you wanted me to change the instructions for all persistent live resources. Creating them on the fly is for CIs/nightlies - not dev testing. Those resources are meant to be persisted, as the old instructions described.

Reply:
I'm fine with doing `setx` on the explicit set of variables we create.  I'm saying we shouldn't `setx` everything on a random user's machine that starts with AZURE.